src/openassetio-python/cmodule/src/managerApi/ManagerInterfaceBinding.cpp

```diff
@@ -97,6 +97,10 @@ struct PyManagerInterface : ManagerInterface {
                            hostSession, successCallback, errorCallback);
   }
 
+  [[nodiscard]] bool hasCapability(ManagerInterface::Capability managerCapability) override {
+    PYBIND11_OVERRIDE_PURE(bool, ManagerInterface, hasCapability, managerCapability);
```

Comment:
[fix] SAM needs an implementation

Reply:
I wonder if we should remove SAM. We have the template manager, and BAL. SAM doesn't even implement `isEntityReferenceString` so is already a non-conforming manager (though at least we can indicate that via `hasCapability`).

Though, since SAM does set `kInfoKey_EntityReferencesMatchPrefix`, I think it will still work, despite not implementing `isEntityReferenceString`, since that'll be short-circuited anyway. So should its `kEntityReferenceIdentification` capability be `true`?

Reply:
Added method in f8961807, returning `False` for everything except `kManagementPolicyQueries`, i.e. assuming that `kInfoKey_EntityReferencesMatchPrefix` is not sufficient for `kEntityReferenceIdentification` capability.

Reply:
Yeah - you should def implement the method too I think 👍